INSTALLATION.md

```diff
@@ -176,42 +178,7 @@ You need to setup your own local instance of [Talawa-API](https://github.com/Pal
 1. You need to first setup the 2 supporting projects locally. Please refer the INSTALLATION.md of the respective repository for further guidance.
    1. [talawa-api](https://github.com/PalisadoesFoundation/talawa-api)
    2. [talawa-admin](https://github.com/PalisadoesFoundation/talawa-admin)
-2. You need to setup your own instance of MongoDB database locally or on cloud.
-
```

Comment:
Change all links that refer to the Talawa-API `develop` branch. These could change without warning.
Just say "refer to the relevant section of the `INSTALLATION.md` guide in the [Talawa-API repo](https://github.com/PalisadoesFoundation/talawa-api)"

Reply:
There are many references to this that you did not edit in the PR